timify-api/app.js

```diff
@@ -29,8 +30,13 @@ async function taddyGraphqlRequest(query, variables) {
       'X-API-KEY': `${process.env.API_KEY}`,
     }
   });
```

Comment:
the formatting for the fields + this end bracket also looks off to me. I'm having a hard time reading the separation of things happening.